.github/workflows/kind-e2e.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
 
         ingress:
         - kourier
-        # - kourier-tls
+        - kourier-tls
```

Comment:
Reverted it so I can test. Tests seem to pass here consistently.

Reply:
https://github.com/knative/serving/issues/15052 should also help.